packages/amplify-environment-parameters/src/environment-parameter-manager.ts

```diff
@@ -48,6 +48,12 @@ export const saveAll = async (serviceUploadHandler?: ServiceUploadHandler): Prom
  * Class for interfacing with environment-specific parameters
  */
 class EnvironmentParameterManager implements IEnvironmentParameterManager {
+  private categoriesThatCannotBeCloned = [
```

Comment:
```suggestion
  private categoriesThatCannotBeClonedAutomatically = [
```

Or something more descriptive. When I read code below I'm under impression that these can be cloned under certain criteria which is examined in logic below.